bio_hansel/const.py

```diff
@@ -48,15 +48,21 @@
 n_tiles_matching_subtype
 n_tiles_matching_subtype_expected
 file_path
+avg_tile_coverage
 qc_status
-qc_message""".strip().split('\n')
+qc_message
+""".strip().split('\n')
+
 
 SIMPLE_SUMMARY_COLS = """
 sample
 subtype
+coverage
 qc_status
 qc_message
 """.strip().split('\n')
 
-REGEX_FASTQ = re.compile(r'^(.+)\.(fastq|fq)(\.gz)?$')
+REGEX_FASTQ = re.compile(r'^(.+)\.(fastq|fq|fastqsanger)(\.gz)?$')
```

Comment:
One thing I found useful when Galaxy tries to append some strange extensions is to force it to be something more standard (just `fastq`) when you're making the symlinks because in Galaxy the extension could be `fastqsolexa` or `fastqillumina` as well as `fastqsanger` at least in the latest version of Galaxy.

Reply:
So within the Galaxy wrapper, it might just be better for me to symlink to a .fastq.

Reply:
Yeah, it seems like you can do whatever you want with the filenames in the Cheetah/Python portion of the tool xml.